tools/decoder/Decoder.cs

```diff
@@ -26,7 +26,6 @@
 using Newtonsoft.Json.Linq;
 using CsProtocol;
 using System.Linq;
-using Fiddler;
```

Comment:
My minor concern is we still need to make sure it works with Fiddler too. I honestly don't remember why we need this using, but if you can manually check that the Fiddler Inspector compiles Okay, that'd be great.

Reply:
@maxgolov , I will check. By Fiddler Inspector do you mean project [OneDSInspector](https://github.com/microsoft/cpp_client_telemetry/tree/main/tools/fiddler/OneDSInspector)?

Reply:
Yup! Thank you so much. If we can't remove that, maybe we can use `#if` or something like that.

Reply:
I think it was added by actually: #1070